runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectGraphVisitorTest.java

```diff
@@ -87,7 +86,7 @@ public void processElement(DoFn<String, String>.ProcessContext c)
   @Test
   public void getRootTransformsContainsRootTransforms() {
     PCollection<String> created = p.apply(Create.of("foo", "bar"));
-    PCollection<Long> counted = p.apply(Read.from(CountingSource.upTo(1234L)));
+    PCollection<Long> counted = p.apply(GenerateSequence.from(0).to(1234L));
```

Comment:
Yes. These changes aren't required for this change, and I'd like to avoid making unnecessary changes where possible.

As an aside, the deprecation doesn't concern me (`CountingSource` shouldn't be used by users as `GenerateSequence` is the abstraction we'd prefer they'd use, but it's acceptable to use in runner tests).